cli/trampolines/trampolines_aarch64.S

```diff
@@ -1,12 +1,27 @@
 #include "../../src/jl_exported_funcs.inc"
 
+// On macOS, we need to prepend underscores on symbols
+#if defined(__APPLE__) && defined(__MACH__)
```

Comment:
When were you running 9.x Classic MacOS userpace, and not using the mach kernel???